install/disconnected_install.adoc

```diff
@@ -453,7 +452,7 @@ For Red Hat support, a {gluster-native} subscription is required for `rhgs3/` im
 [source, bash]
 ----
 $ docker save -o ose3-logging-metrics-images.tar \
-    registry.access.redhat.com/openshift3/logging-auth-proxy \
+    registry.access.redhat.com/openshift3/logging-oauth-proxy \
```

Comment:
logging-auth-proxy  is replaced with oauth-proxy.

Reply:
Fixed. Not sure how that happened! Thanks!